bigflow/commons.py

```diff
@@ -24,23 +21,6 @@ def run_process(cmd, **kwargs):
     return result_output
 
 
-def resolve(path: Path):
```

Comment:
It's a part of the API visible in the docs, scaffold, our source code. If you want to remove it (in another PR), do it by:

* updating examples from docs
* updating scaffold
* updating out in-house starter
* adding "deprecated" message
* ...?

Reply:
Added back 'resolve' with deprecation warning. Updated docs/scaffold.